apps/nouns-camp/src/components/proposal-candidate-screen.js

```diff
@@ -157,45 +155,44 @@ const ProposalCandidateScreenContent = ({
   });
 
   const [
-    pendingRepliesByTargetFeedItemId,
-    setPendingRepliesByTargetFeedItemId,
-  ] = React.useState(() => {
-    const initialReplyTargetId = searchParams.get("reply-target");
-    if (initialReplyTargetId == null) return {};
-    return { [initialReplyTargetId]: "" };
-  });
-  const setReply = React.useCallback((targetFeedItemId, reply) => {
-    setPendingRepliesByTargetFeedItemId((s) => ({
-      ...s,
-      [targetFeedItemId]: reply,
-    }));
-  }, []);
-  const [pendingReplyTargetFeedItemIds, setPendingReplyTargetFeedItemIds] =
-    React.useState(() => Object.keys(pendingRepliesByTargetFeedItemId));
-  const [pendingRepostTargetFeedItemIds, setPendingRepostTargetFeedItemIds] =
-    React.useState(() => {
-      const initialRepostTargetId = searchParams.get("repost-target");
-      if (initialRepostTargetId == null) return [];
-      return [initialRepostTargetId];
-    });
+    {
+      comment: pendingComment,
+      support: pendingSupport,
+      replies: pendingReplies,
+      reposts: pendingReposts,
+    },
+    {
+      setComment: setPendingComment,
+      setSupport: setPendingSupport,
+      addReply,
+      setReply,
+      deleteReply,
+      addRepost,
+      deleteRepost,
+      clearPost,
+    },
+  ] = useCachedPost(`vwr:c:${candidateId}`, { searchParams });
```

Comment:
ugh unsure about this format, but just wanted a simple way to split prop/candie votes.  any good alternatives welcome